pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -462,8 +485,32 @@ func doInstall(ctx *zedmanagerContext, uuidStr string,
 				changed = true
 				continue
 			}
+
 			ss.MissingDatastore = false
-			AddOrRefcountDownloaderConfig(ctx, safename, ss, dst)
+			downloadUrl := dst.Fqdn
+			if len(strings.TrimSpace(dst.Dpath)) > 0 {
+				downloadUrl += "/" + dst.Dpath
+			} else {
+				log.Debugf("doInstall: empty Dpath")
+			}
+			if len(strings.TrimSpace(ss.Name)) > 0 {
+				downloadUrl += "/" + ss.Name
+			} else {
+				log.Debugf("doInstall: empty ss.Name")
+			}
+			log.Infof("doInstall: downloadUrl: %s", downloadUrl)
+
+			if ss.Format == "8" {
```

Comment:
what is 8

Reply:
8 is format container. This really should be enum instead of a string. Was thinking of making that change separately.